sdk/scheduler/build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id 'de.fuerstenau.buildconfig' version '1.1.8'
-    id 'com.github.johnrengelman.shadow' version '2.0.3'
+    id 'com.github.johnrengelman.shadow' version '5.0.0'
```

Comment:
This is needed to for the shadow plugin to work with gradle. Our release scripts are broken currently because of this.